README.md

```diff
@@ -340,16 +73,16 @@ CONDITIONS OF ANY KIND, either express or implied. See the License for the
 specific language governing permissions and limitations under the License.
 
 <!-- references -->
-[consul]: https://consul.io/
-[es]: https://twitter.github.io/effectivescala/
+[coc]: wiki/Linkerd-code-of-conduct
+[cncf]: https://www.cncf.io/about
 [finagle]: https://twitter.github.io/finagle/
-[funsuite]: http://www.scalatest.org/getting_started_with_fun_suite
 [k8s]: https://k8s.io/
-[l5d-ci]: https://circleci.com/gh/linkerd/linkerd
```

Comment:
Removing the l5d-ci anchor breaks the link on the circle ci badge at the top of this document.